<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form</class>
 <widget class="QDialog" name="form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>353</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>查找帧定义</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/new/prefix1/icon/view.png</normaloff>:/new/prefix1/icon/view.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_12">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_11">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="2">
           <widget class="QLineEdit" name="lineEdit">
            <property name="text">
             <string>AA 55</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBox_2">
            <property name="text">
             <string>帧长</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QWidget" name="widget_3" native="true">
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <widget class="QRadioButton" name="radioButton">
                 <property name="text">
                  <string>定长</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="1" colspan="3">
                <widget class="QLineEdit" name="lineEdit_2">
                 <property name="text">
                  <string>5</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0" colspan="2">
                <widget class="QRadioButton" name="radioButton_2">
                 <property name="text">
                  <string>变长</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>偏移字节</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="3">
                <widget class="QLineEdit" name="lineEdit_3">
                 <property name="placeholderText">
                  <string>从0开始</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>数据类型</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="3">
                <widget class="QComboBox" name="comboBox">
                 <item>
                  <property name="text">
                   <string>无符号单字节</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>有符号单字节</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>无符号双字节</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>有符号双字节</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>无符号四字节</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>有符号四字节</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>无符号八字节</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>有符号八字节</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>帧头</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBox_3">
            <property name="text">
             <string>自定义</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QWidget" name="widget_4" native="true">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_5">
               <item row="0" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>偏移字节</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="lineEdit_4">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="placeholderText">
                  <string>从0开始</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QPushButton" name="pushButton_3">
                 <property name="text">
                  <string>添加</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>对应字节</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="lineEdit_5"/>
               </item>
               <item row="1" column="2">
                <widget class="QPushButton" name="pushButton_4">
                 <property name="text">
                  <string>删除</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QTableWidget" name="tableWidget">
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_10">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>确定</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButton_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>取消</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pushButton_5">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>监视</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
